File: tests/c/TestBridge.c
#define main pattern_main
#include "../../oop_in_c/Bridge/Bridge.c"
#undef main
#include <stdio.h>

int main() {
  printf("Testing C Bridge...\n");
  // Call pattern main
  pattern_main();
  printf("C Bridge Test Passed!\n");
  return 0;
}

File: tests/java/TestBridge.java
public class TestBridge {
  public static void main(String[] args) {
    System.out.println("Testing Java Bridge...");
    try {
      Client.main(args);
    } catch (Exception e) {
      e.printStackTrace();
      System.exit(1);
    }
    System.out.println("Java Bridge Test Passed!");
  }
}
